test/Pass/nop2.ll

```diff
@@ -31,5 +31,5 @@ jmp:
 declare i32 @a(i32) local_unnamed_addr #1
 declare i1 @branch(i32) local_unnamed_addr #1
 
-; CHECK: 3 souper - Number of instructions replaced by another instruction
-; XFAIL: *
```

Comment:
Reenable this test case, but I don't know why one of the replacement is not being counted in stats. The result .ll file is showing that the replacement does take place. I'll figure it out soon.